server/src/main/java/org/elasticsearch/health/node/action/TransportHealthNodeAction.java

```diff
@@ -38,10 +40,22 @@
 
     private static final Logger logger = LogManager.getLogger(TransportHealthNodeAction.class);
 
+    /**
+     * This is the amount of time given as the timeout for transport requests to the health node. The default is fairly low because these
+     * actions are expected to be lightweight and time-sensitive.
+     */
+    public static final Setting<TimeValue> HEALTH_NODE_TRANSPORT_ACTION_TIMEOUT = Setting.timeSetting(
+        "health_node.transport_action_timeout",
+        TimeValue.timeValueSeconds(5),
+        TimeValue.timeValueMillis(1),
+        Setting.Property.NodeScope
```

Comment:
I just want to test my understanding, this setting is not dynamically updated that's why we do not have a listener for this, right?

Reply:
It is not dynamically updatable as I've written it. It didn't seem worth the effort to do that here since this one will probably never even be touched. What do you think?